Platform/ApollolakeBoardPkg/BoardConfig.py

```diff
@@ -152,10 +152,8 @@ def GetDscLibrarys (self, BuildPkgName = "BootLoaderCorePkg"):
 			'ShellExtensionLib|Platform/$(BOARD_PKG_NAME)/Library/ShellExtensionLib/ShellExtensionLib.inf',
 			'BootMediaLib|Silicon/ApollolakePkg/Library/BootMediaLib/BootMediaLib.inf',
 			'FlashDescriptorLib|Silicon/ApollolakePkg/Library/FlashDescriptorLib/FlashDescriptorLib.inf',
+			'VTdLib|Silicon/$(SILICON_PKG_NAME)/Library/VTdLib/VTdLib.inf'
```

Comment:
Why VTd ('T') is uppercase in naming? Maybe clean it up too?

Reply:
It's VT-d actually. I think changing to Vtd would require directory name change as well, so not that much benefit.

Reply:
The camel case really not very expressive but I understand now.

Reply:
According to EDKII coding style,  it should be VtdLib and VtdLib.inf.  :)

Reply:
Okay, coding convention. Let me change whole VTd to Vtd including directories.